test/integration/model.test.js

```diff
@@ -233,7 +233,8 @@ describe(Support.getTestDialectTeaser('Model'), () => {
       expect(user.deletedAtThisTime).to.exist;
     });
 
-    it('returns proper defaultValues after save when setter is set', async function() {
+    // The Oracle dialect doesn't support empty string in a non-null column
+    (dialect !== 'oracle' ? it : it.skip)('returns proper defaultValues after save when setter is set', async function() {
```

Comment:
You can change the test to use a different default value

Reply:
Changed the default value to 'A' for the Oracle dialect